azure-pipelines-deploy.yml

```diff
@@ -0,0 +1,65 @@
+trigger:
+  tags:
+    include:
+      - "*"
```

Comment:
Trying this to get it to only build when we push a release tag, but not sure that actually works. Can also leave the job disabled in the azure dashboard until we're ready to do an actual release.

Reply:
Going to switch it to `trigger: none` for now and we can revisit once we've spent more time with it